turbinia/turbiniactl.py

```diff
@@ -463,7 +463,10 @@ def process_args(args):
   # file explicitly set on the command line) to set a file-handler now that we
   # have the logfile path from the config.
   if server_flags_set or worker_flags_set or args.log_file:
-    logger.setup()
+    if args.log_file:
+      logger.setup(user_specified_log=True)
```

Comment:
Instead of overriding `LOG_DIR` with a different data type (file vs. dir), what do you think about just passing in the user specified logfile here instead of setting a boolean?  Is `LOG_DIR` used elsewhere that we would need access to that file name?

Reply:
Yes sounds good, I went ahead and changed the variable parameter to `log_file_path` and I'm passing `user_specified_log` into `log_file_path` if `args.log_file` is present. Looking back, not seeing it used elsewhere where we'd need access to the variable